### ISSUE TITLE
Fewer meetings

### ISSUE THREAD

**Issue by nottrobin** [canonical-webteam/practices#135]
Lifted directly from [@pmahnke's slide](https://docs.google.com/presentation/d/1s7A-Pnv4XjBZdH33yJvd06E4XSKxs6tVoQXV3TX90UU/edit#slide=id.g499960e33f_0_94):

> - Try to book meetings in the afternoons - keep mornings for work
>   - Please re-do the calendar
>   - Move meetings out of Babbage, Cerf, Dijkstra
> - Only have a meeting if really you need one
> - For the invite
>   - Only invite who you need
>   - ACCEPT or DECLINE the meeting
>   - Check that required people can attend
>   - Add a video link and a description
>   - Make the meeting MODIFIABLE  
> - And remember, meetings don’t have to last the full amount of time

Someone please convert this into a practice document. Maybe @matthewpaulthomas would love to do this? =)

**Comment by nottrobin** [comment]
Other points:

- Meetings being modifiable by others by default is a setting - turn it on
- Use the "Out of office" event type in Calendar, which will auto-decline any overlapping meetings

**Comment by nottrobin** [comment]
Yes I think things may have moved on since the "try to book meetings in the afternoons" point - although it did obviously originate with @pmahnke. But @matthewpaulthomas if you do end up writing this document, feel free to leave that one out and then we can see if it comes up in the inevitable PR discussion. The same is true of any other point - feel free to write the document as you think it should be written.

I do agree that the only way for these practices to actually materialise is for team members (yes, especially managers) to be constantly reminding other team members. However, I'd say this is true of every possible practice that we could write down. The reason for these practices to exist is to declare them as formal team standards precisely so that we have an authoritative document to point to when reminding each other - so we have a formal record of what we previously decided and we don't have a long protracted discussion with someone new.

So personally I think a practice document along these lines would have as much value as any other, and more than many.

**Comment by hatched** [comment]
Keep in mind that us in North America can't reliably make EU morning meetings because of the TZ.

**Comment by nottrobin** [comment]
@hatched yes I think that was part of the rationale behind the original suggestion. So maybe clarification from @pmahnke might help?

**Comment by pmahnke** [comment]
Just to clarify.  The idea of afternoon meetings was because we surveyed the team and looked at research and it appeared that mornings were a better time for heads down work.  The blocked afternoons were because after we arranged all the afternoon meetings, MS asked Carla and me to keep these days free (annoyingly); however, I have removed these now.  I still think afternoon meetings are better in most situations, but it isn't a hard rule.

I would also add a section on cancelling meetings by remembering to check if it is still needed well ahead of time, freeing up people's time and the room in advance is a gift that everyone appreciates.  Also, resisting the desire to create recurring meetings when a series of one-off meetings might suffice.  And finally, attempting to not use computers and phones in meetings unless demoing or taking notes would be a good goal.
